app/uk/gov/hmrc/serviceconfigs/service/ConfigService.scala

```diff
@@ -243,12 +243,22 @@ class ConfigService @Inject()(
 
   def appConfig(serviceName: String)(implicit hc: HeaderCarrier): Future[Seq[ConfigSourceEntries]] =
```

Comment:
can we remove this variant and the endpoint now? It was added for the outbound routes, which has now been reworked.